azure-pipelines.yml

```diff
@@ -107,20 +107,20 @@ jobs:
         inputs:
           codeCoverageTool: Cobertura
           summaryFileLocation: '$(System.DefaultWorkingDirectory)/qcodes/coverage.xml'
-      - script: |
-          CALL C:\Miniconda\condabin\conda_hook.bat
-          CALL conda activate qcodes
-          cd docs
-          REM Turn warnings into errors
-          set SPHINXOPTS=-W -v
```

Comment:
I think we should leave this block as a cmd or powerscript. We need to test the make.bat file and that cannot run in bash

Reply:
what if i do `./make.bat`? according to google search it will execute the bat file in cmd

Reply:
I assume you mean bash and not cmd? Is set valid in bash I would assume not

Reply:
ah, sorry, i completely misunderstood your comment, i thought you were referring to `make.bat` call.

indeed, instead of `set` it should be `export`.

i hope bash will tell me that i'm stupid.. :)

Reply:
I am referring to the make.bat call. .bat files are windows batch scripts they are intended to run in cmd(aka batch) and not in bash. You are right however that in git bash on window you can run a .bat script if you do `./make.bat` even if you are in bash 

in cmd `./make.bat` is a syntax error

Reply:
ok. decided to revert back to script for simplicity